pkg/graph/resolvers/trb_request_system.go

```diff
@@ -0,0 +1,35 @@
+package resolvers
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/dataloaders"
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+// TRBRequestSystems utilizies a dataloader to retrieve systems linked to a given system intake ID
```

Comment:
`system intake` -> `trb request`

Reply:
this one here and then the few in other comments

Reply:
Addressed in [ca20523](https://github.com/CMSgov/easi-app/pull/2445/commits/ca20523b290c8a45960d581688769f1feb3c66a8)